ExampleMod/Content/Items/Consumables/ExampleLifeFruit.cs

```diff
@@ -65,6 +67,8 @@ public class ExampleLifeFruitPlayer : ModPlayer
 		public int exampleLifeFruits;
 
 		public override void ResetEffects() {
+			//Increasing health in the ResetEffects hook in particular is important so it shows up properly in the player select menu
+			//and so that life regeneration properly scales with the bonus health
```

Comment:
```suggestion
			// Increasing health in the ResetEffects hook in particular is important so it shows up properly in the player select menu
			// and so that life regeneration properly scales with the bonus health
```